src/client/rendering-unit.h

```diff
@@ -53,6 +57,8 @@ class RenderingUnit final : public IResource {
   const uint64_t id_;
   AtomicCommandQueue *update_rendering_queue_;
 
+  std::weak_ptr<GlBaseTechnique> gl_base_technique_;
+
```

Comment:
This seems one of pending_ state, and we need to move this to rendering state when commited.